docs/releases/4.0.txt

```diff
@@ -58,6 +58,45 @@ For example::
 Functional unique constraints are added to models using the
 :attr:`Meta.constraints <django.db.models.Options.constraints>` option.
 
+
+Supoort SQL COMMENT
```

Comment:
I'm not sure this is enough to count as a major feature IMO (other opinions may vary) - if it does, I think we don't need the note about ManyToManyField here, just the first example is enough I think.

Reply:
Not sure what makes a feature "major" vs. "minor", but I'd like to share what I think makes this feature _at least notable_:

- This feature is among the first Django features targeted primarily at database administrators, data analysts, data scientists, and others who are looking to consume data that is managed by Django.
- The idea for this feature is 11 years old (if you go back to ticket #13867 which is the predecessor for ticket #18468 that this patch will close).

I'm not a committer so I'll defer to committers on what the "bar" is for "major feature" status...

Reply:
I agree that it's not a major feature but I think at least it needs to be mentioned because it is a new feature.
I am not familiar with django's policy, so if it seems inappropriate for releases note to comment here, please leave one more comment.